src/cmds/inpost.ts

```diff
@@ -0,0 +1,59 @@
+import FluorineClient from '@classes/Client';
+import Embed from '@classes/Embed';
+import { Message } from 'discord.js';
+import { fetch } from 'undici';
+export async function run(
+    client: FluorineClient,
+    message: Message,
+    args: string[]
+) {
+    if (!args[0]) {
+        return message.reply(
+            client.language.get(message.guild.preferredLocale, 'INPOST_ARGS')
+        );
+    }
+    let statusURL;
+    if (message.guild.preferredLocale === 'pl') {
+        statusURL = 'https://api-shipx-pl.easypack24.net/v1/statuses';
+    } else {
+        statusURL = 'https://api-shipx-pl.easypack24.net/v1/statuses?lang=en';
+    }
```

Comment:
can we make this a string in the locales json?

Reply:
we can but ummmmmmmmmmmmmmmm yeah

Reply:
we could use ternary operators to shorten that

Reply:
just add it as a locale string, it's easier and otherwise it will break if we ever add more languages